Dockerfile.cli.jdk8

```diff
@@ -0,0 +1,53 @@
+# This image will be published as dspace/dspace-cli
+# See https://dspace-labs.github.io/DSpace-Docker-Images/ for usage details
+#
+# This version is JDK8 compatible
+# - openjdk:8-jdk
+# - ANT 1.10.5
+# - maven:3-jdk-8
+# - note:
+# - default tag for branch: dspace/dspace-cli: dspace/dspace-cli:dspace-6_x
+
+# Step 1 - Run Maven Build
+FROM dspace/dspace-dependencies:dspace-6_x as build
+ARG TARGET_DIR=dspace-installer
+WORKDIR /app
+
+# The dspace-install directory will be written to /install
+RUN mkdir /install \
+    && chown -Rv dspace: /install
+
+USER dspace
+
+# Copy the DSpace source code into the workdir (excluding .dockerignore contents)
+ADD --chown=dspace . /app/
+COPY dspace/src/main/docker/local.cfg /app/local.cfg
+
+# Build DSpace.  Copy the dspace-install directory to /install.  Clean up the build to keep the docker image small
+RUN mvn package -Dmirage2.on=true && \
+  mv /app/dspace/target/${TARGET_DIR}/* /install && \
+  mvn clean
+
+# Step 2 - Run Ant Deploy
+FROM openjdk:8-jdk as ant_build
+ARG TARGET_DIR=dspace-installer
+COPY --from=build /install /dspace-src
+WORKDIR /dspace-src
+
+# Create the initial install deployment using ANT
+ENV ANT_VERSION 1.10.5
+ENV ANT_HOME /tmp/ant-$ANT_VERSION
+ENV PATH $ANT_HOME/bin:$PATH
+
+RUN mkdir $ANT_HOME && \
+    wget -qO- "https://www.apache.org/dist/ant/binaries/apache-ant-$ANT_VERSION-bin.tar.gz" | tar -zx --strip-components=1 -C $ANT_HOME
+
+RUN ant init_installation update_configs update_code
+
+# Step 3 - Run tomcat
```

Comment:
That comment looks like a copy and paste error to me. We do not run tomcat in this image.